applications/StructuralMechanicsApplication/custom_constitutive/constitutive_laws_integrators/high_cycle_fatigue_law_integrator.h

```diff
@@ -0,0 +1,320 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Sergio Jimenez/Alejandro Cornejo/Lucia Barbu
+//
+
+#if !defined(KRATOS_HIGH_CYCLE_FATIGUE_LAW_INTEGRATOR_H_INCLUDED)
+#define KRATOS_HIGH_CYCLE_FATIGUE_LAW_INTEGRATOR_H_INCLUDED
+
+// System includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/checks.h"
+#include "includes/serializer.h"
+#include "includes/properties.h"
+#include "utilities/math_utils.h"
```

Comment:
I am quite sure you don't need some of those, please check and remove the unused ones